web/src/pages/contact.tsx

```diff
@@ -1,15 +1,96 @@
-import { Link } from 'gatsby';
+import { Box, Flex, Heading, Label, Link } from '@theme-ui/components';
 import React from 'react';
+import { SystemStyleObject } from 'theme-ui';
+import MapSvg from '../assets/map.svg';
+import { EmployeeList } from '../components/contact/exployee-list';
 import { Container } from '../components/container';
 import { Layout } from '../components/layouts';
 
+const styles: SystemStyleObject = {
+  '.image': {
+    width: '396px',
+    height: '328px',
+    right: '180px',
+    position: 'absolute',
+    top: '200px',
+  },
+  '.mobileImage': {
+    width: '40%',
+    position: 'absolute',
+    display: 'none',
+    right: '-13%',
+  },
+  ['@media screen and (max-width: 960px)']: {
+    '.image': {
+      display: 'none',
+    },
+    '.mobileImage': {
+      display: 'block',
+    },
```

Comment:
I've given a few examples how to remove this `@media` block. You should never have to assign media queries when using a style system like theme-ui.